pkg/workloads/lib/package.py

```diff
@@ -35,6 +36,20 @@ def get_build_order(python_packages):
     return build_order + sorted([name for name in python_packages if name != "requirements.txt"])
 
 
+def get_restricted_packages():
+    cortex_packages = {"pyspark": "2.4.0", "tensorflow": "1.12.0"}
+    req_files = glob.glob("/src/**/requirements.txt", recursive=True)
+    for req_file in req_files:
+        with open(req_file) as f:
+            lines = [line.strip() for line in f.readlines()]
+            for line in lines:
+                print(line)
```

Comment:
We might as well support whole-line comments, e.g. continue if `line.startswith("#")`